conductr_cli/bndl_create.py

```diff
@@ -139,21 +145,13 @@ def bndl_create(args):
                 return 2
 
             input_dir = find_bundle_conf_dir(temp_dir)
-            bundle_conf_path = '' if not input_dir else os.path.join(input_dir, 'bundle.conf')
 
-            if not input_dir or not os.path.exists(bundle_conf_path):
```

Comment:
This abortion has been removed given the fact that we now can create an output without any input or with an input that does not contain a `bundle.conf`.